docs/engineering/wsrtc/wsrtc-quickstart.md

```diff
@@ -0,0 +1,97 @@
+---
+id: wsrtc-quickstart
+title: WSRTC Quick Start
+---
+
+# WSRTC - Quick Start Guide
+
+## Quick Install
+
+Clone and run:
+
+
+```bash
+
+# Clone this repository
+
+git clone https://github.com/webaverse/wsrtc.git
+
+# Go into the repository
+
+cd wsrtc/
+
+```
+Install the dependencies:
+
+```bash
+
+# Install dependencies
+
+$ npm install
```

Comment:
90% of these docs are these same things over and over and over.

How many times do we need to tell people how to use npm and git?

Reply:
Yeah. ken broke out that one setup document and then refers to it in every other document. I was thinking we could probably do the same with alot of the rest of that.

Reply:
I agree. The docs should be written thoughtfully and empathically, by humans for humans, rather than an exercise in copypasta.